server/routes/user.ts

```diff
@@ -265,7 +265,11 @@ router.post('/import-from-plex', async (req, res, next) => {
         await userRepository.save(user);
       } else {
         // Check to make sure it's a real account
-        if (account.email && account.username) {
+        if (
+          account.email &&
+          account.username &&
+          (await mainPlexTv.checkUserAccess(Number(account.id)))
```

Comment:
Will this not throw and break the loop? Preventing users from being added?

Reply:
Wouldn't it only throw if Plex API access is unavailable, in which case we wouldn't want to import users?

Reply:
I mean, we can add a `try`/`catch` here but I feel like exiting is likely the desired behavior?  If the check throws once, isn't it likely to throw for any following user checks as well?

Reply:
checkUserAccess throws when it can't find the user in the list. Not just when it can't connect to the api.

Reply:
Ahh I see, I'll fix that!